test/layers/domain/get_book_details_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mibook/layers/domain/repository/search_repository.dart';
+import 'package:mibook/layers/domain/usecases/get_book_details.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'fakes/fake_book_domain.dart';
+
+@GenerateNiceMocks([MockSpec<ISearchRepository>()])
+import 'get_book_details_test.mocks.dart';
+
+void main() {
+  late MockISearchRepository mockSearchRepository;
```

Comment:
Could we avoid `late` variables in the tests? That could provoke crashes in our CI.